WikiTreeAPI.js

```diff
@@ -209,24 +215,28 @@ WikiTreeAPI.getRelatives = function (IDs, fields, options = {}) {
     });
 };
 
-// This is just a wrapper for the Ajax call, sending along necessary options for the WikiTree API.
-WikiTreeAPI.postToAPI = function (postData) {
-    var API_URL = "https://api.wikitree.com/api.php";
-
-    var ajax = $.ajax({
-        // The WikiTree API endpoint
-        url: API_URL,
-
-        // We tell the browser to send any cookie credentials we might have (in case we authenticated).
-        xhrFields: { withCredentials: true },
-
-        // We're POSTing the data, so we don't worry about URL size limits and want JSON back.
-        type: "POST",
-        dataType: "json",
-        data: postData,
-    });
+// This is just a wrapper for the JavaScript fetch() call, sending along necessary options for the WikiTree API.
+WikiTreeAPI.postToAPI = async function (postData) {
+    const API_URL = "https://api.wikitree.com/api.php";
 
-    return ajax;
+    let formData = new FormData();
+    for (var key in postData) {
+        formData.append(key, postData[key]);
+    }
+    // We're POSTing the data, so we don't worry about URL size limits and want JSON back.
+    let options = {
+        method: "POST",
+        credentials: localTesting ? "omit" : "include",
+        headers: {
+            "Content-Type": "application/x-www-form-urlencoded",
```

Comment:
is it ok to send this one? (`application/x-www-form-urlencoded`)

wouldn't it be better to send `application/json`?

But, okay, current solution also sends `application/x-www-form-urlencoded` and receives `application/json`.

![image](https://user-images.githubusercontent.com/20949562/198041860-4a6c347d-d6be-4c12-ad15-d1d965406ed6.png)

Reply:
I have to admit that when I originally implemented this in my couples-view branch 3 months ago, I came to this setting by trial and error - I could not get `application/json` to work (if I remember correctly), but maybe I did something else wrong (e.g. not setting up the FormData correctly).

Reply:
Maybe it's implemented that way that it only accepts `application/x-www-form-urlencoded`, unfortunately it's not specified in API docs.

Reply:
The current API code is built on some code that was inherited from an old base. It's expecting the key/value pairs in the query string or POST data. I'm not sure what it would do with a Content-Type of application/json. I've added investigation of that to my queue.

For now, let's keep the Content-Type as is. It works adequately. We can return to this after I've checked through the server side of things.